plugins/InspectionGadgets/InspectionGadgetsAnalysis/src/com/siyeh/ig/style/UnnecessaryConstructorInspection.java

```diff
@@ -119,7 +118,7 @@ public void visitClass(@NotNull PsiClass aClass) {
       final PsiCodeBlock body = constructor.getBody();
       if (ControlFlowUtils.isEmptyCodeBlock(body) ||
           isSuperConstructorInvocationWithoutArguments(ControlFlowUtils.getOnlyStatementInBlock(body))) {
-        registerMethodError(constructor);
+        registerError(constructor, ProblemHighlightType.LIKE_UNUSED_SYMBOL);
```

Comment:
The unused method inspection only highlights the identifier of the method and not the body. I wonder if we should do the same here for consistency?

Reply:
I think I will be able to improve this in another PR. I will highlight the whole constructor and cite only the identifier in the message but this would be too much changes for this PR.